### ISSUE TITLE
Paid AV Instructions

### ISSUE THREAD

**Issue by tldale** [saferwall/saferwall#282]
Hello,

Please provide a list of where you can obtain the AV licenses for the non-free AVs. Also please then explain how we can create those containers since we are unable to pull them from the cloud.

Thank you!

**Comment by kscieslinski** [comment]
@actuallyachraf is there a reason why this issue has been closed? The part regarding the AV licenses would be useful as at the moment I cannot find any docs describing what products saferwall uses (AV companies tend to have multiple different products and at the moment one has to retrieve this information from source code).

**Comment by LordNoteworthy** [comment]
I agree @actuallyachraf, this is still pending on my side, let's keep this one open.

**Comment by actuallyachraf** [comment]
The issue wasn't tagged and mistakenly assumed it was fixed, we'll try and setup a public document maybe /AV.md that would contain antivirus related instructions (paid licenses, self-hosted setups...)
You're welcome to jump on to our Discord !

**Comment by kscieslinski** [comment]
Hi, is there any update on this? ;)

**Comment by actuallyachraf** [comment]
bumping this up for @LordNoteworthy paid AV instructions will be up here EOW https://github.com/saferwall/saferwall-docs

**Comment by analyserdmz** [comment]
Bumping up due to no instructions yet. That would be very useful.
